src/lib.rs

```diff
@@ -55,6 +55,6 @@ pub(crate) mod segment;
 #[doc(hidden)]
 pub mod test_utils;
 #[macro_use]
-mod util;
+pub mod util;
```

Comment:
We probably want to hide this module in document. Like `#[doc(hidden)]`

Reply:
done.